### PR TITLE
Upgrade cloudWatchInsightsAddOn in Observability Builder

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by shapirov103 — COMMENTED** [review]
please add version increment to package.json. 

Also, please confirm that you validated it in CDK observability accelerator manually - this is a prereq for merging.

File: lib/builders/observability-builder.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export class ObservabilityBuilder extends BlueprintBuilder {
     }
 
     public withContainerInsightProps(props: addons.ContainerInsightAddonProps) : this {
```
[the remark this answers is not in the document]
Comment (elamaran11) [reply lib/builders/observability-builder.ts]: Ya this is fixed. I never tested it so missed it.

**Comment by elamaran11** [comment]
> please add version increment to package.json.
> 
> Also, please confirm that you validated it in CDK observability accelerator manually - this is a prereq for merging.

Done, Everything works fine, Please feel free to merge.

**Comment by elamaran11** [comment]
Validation with CDK accelerator is also complete, Please merge.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**